src/donationswap.py

```diff
@@ -265,17 +272,25 @@ def _delete_unconfirmed_offers(self):
 		return count
 
 	def _send_mail_about_expired_offer(self, offer):
+		newExpirey = offer.expires_ts + (offer.expires_ts - offer.created_ts)
```

Comment:
expiry, not expirey. (oh, you kiwis... ;-) )

Reply:
english is hard 🙈